createdatabase.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE community_orgs (
+    id INT PRIMARY KEY,
+    community_org_name VARCHAR(40) UNIQUE NOT NULL,
+    telephone VARCHAR(15),
+    email VARCHAR(50) UNIQUE,
+    address VARCHAR(70),
+    longitude DECIMAL,
+    latitude DECIMAL
+);
+
+CREATE TABLE categories (
+    id INT PRIMARY KEY,
+    category_name VARCHAR(50) UNIQUE NOT NULL
+);
+
+CREATE TABLE community_org_categories (
+    community_org_id INT,
+    categories_id INT
```

Comment:
you need to tell MariaDB that these are foreign keys, and not just static integers (so the database knows it's a relation)

the syntax for this is here
https://mariadb.com/kb/en/constraint/#foreign-key-constraints

more info on foreign keys
https://mariadb.com/kb/en/foreign-keys/
https://www.w3schools.com/sql/sql_foreignkey.asp

e.g. `CONSTRAINT `community_org_constraint```FOREIGN KEY (`community_org_id`) REFERENCES `community_orgs` (`id`) ON UPDATE CASCADE ON DELETE CASCADE```

ON UPDATE tells the database what to do when the **community_organisation** is updated - i.e. here "when the community organisation is updated, update me as well"
ON DELETE is the same, but for deletes

this page explains the different options for ON UPDATE/DELETE: https://mariadb.com/kb/en/foreign-keys/

Reply:
oh! and also this table still needs to have a primary key, even though it is modelling the relationship between the two

a primary key doesn't necessarily have to be an integer like the others, just anything unique (it could be a combination of the two foreign keys here if you like `primary key(idTable1, idTable2),`).. but if you prefer it another `AUTO_INCREMENT` integer would be fine here 😄